packages/ui/components/ui/tx/actions-views/ibc-relay.tsx

```diff
@@ -140,5 +232,23 @@ export const IbcRelayComponent = ({ value }: { value: IbcRelay }) => {
     return <UpdateClientComponent update={update} />;
   }
 
+  if (value.rawAction?.is(MsgTimeout.typeName)) {
+    const timeout = new MsgTimeout();
+    value.rawAction.unpackTo(timeout);
```

Comment:
suggestion: the proto definition of `rawAction` is `Any`. could there potentially be errors when unpacking? It might be a good idea to wrap the [unpackTo](https://github.com/penumbra-zone/web/pull/1810/files#diff-b9fb2f34ee8a60d23b71b17c7677f6248bdf75115b3f05d8699b7b7828adae6eR237) method in a try-catch block when deserializing the rawAction into a message type.

Reply:
Given the conditional block it's in, it _should_ unbpack fine. However, we want this to be fault-tolerant for unexpected values or deserialization errors. Will wrap in try/catch.